scripts/kvmd-bootconfig

```diff
@@ -48,6 +48,13 @@ rw
 # ========== First boot configuration ==========
 
 if [ -n "$FIRSTBOOT$FIRST_BOOT" ]; then
+    ### stop regular kvmd-oled service and show first time setup status in oled
+    systemctl stop kvmd-oled
+    /usr/bin/kvmd-oled --interval=0 --text="First time setup\nDo NOT interrupt\nPlease Wait...\n"
+    
+    ### force reboot after first boot
```

Comment:
Why?

Reply:
If you don't temporarily stop kvmd-oled service, the message will get clobbered.

Reply:
I mean why you need a reboot here?